debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -2157,7 +2157,239 @@ access_driver_type
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_ERROROPT ( '=' | ':' ) (
+        error_element | ('[' error_element ( ',' error_element)* ']' ))
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_FIELDS ( '=' | ':' ) (
+        field_spec et_data_type
+        ( COMMENT col_comment )? (',' COMMENT col_comment )*? )
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_FILEFORMAT ( '=' | ':' ) (
+        SEQUENCEFILE
+        | TEXTFILE
+        | RCFILE
+        | ORC
+        | PARQUET
+        | INPUTFORMAT quoted_string OUTPUTFORMAT quoted_string )
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_EXEC ( '=' | ':' ) (
+        et_directory_spec? et_file_spec )
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_QC ( '=' | ':' ) (
+        et_directory_spec? et_file_spec )
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_OVERFLOW ( '=' | ':' ) (
+        overflow_element | ('[' overflow_element ( ',' overflow_element)* ']'))
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_ROWFORMAT ( '=' | ':' ) (
+        DELIMITED?
+        ( FIELDS TERMINATED BY CHARACTER ( ESCAPED BY CHARACTER )
+        | COLLECTION ITEMS TERMINATED BY CHARACTER
+        | MAP KEYS TERMINATED BY CHARACTER
+        | LINES TERMINATED BY CHARACTER
+        | NULL_ DEFINED AS CHARACTER
+        )
+        | SERDE quoted_string ( WITH SERDEPROPERTIES ( quoted_string '=' quoted_string ( ',' quoted_string '=' quoted_string)*))? )
+    | COM_ORACLE_HDFS_HIVE_BIGDATA_TABLENAME ( '=' | ':' ) (
+        ( DEFAULT | quoted_string ) '.' quoted_string )
+    ;
+
+colmap_entry
+    : LEFT_CURLY_PAREN ET_COL ':' quoted_string ',' ET_FIELD ':' quoted_string RIGHT_CURLY_PAREN
+    ;
+
+error_element
+    : LEFT_CURLY_PAREN ET_ACTION ':' ( ET_REJECT | ET_SETNULL | ET_REPLACE) (',' ET_VALUE ':' quoted_string )
+      ( ',' ET_COL ':' ( quoted_string | '[' quoted_string ( ',' quoted_string )* ']' ))? RIGHT_CURLY_PAREN
+    ;
+
+col_comment
+    : quoted_string
+    ;
+
+overflow_element
+    : LEFT_CURLY_PAREN ET_ACTION ':' ( ET_TRUNCATE | ET_ERROR )
+      ( ',' ET_COL ':' ( quoted_string | '[' quoted_string ( ',' quoted_string )* ']' ))? RIGHT_CURLY_PAREN
+    ;
+
+et_data_type
+    : ( et_primitive_type
+        ARRAY '<' et_data_type '>'
+        MAP '<' et_primitive_type ',' et_data_type '>'
+        STRUCT '<' field_spec et_data_type ( COMMENT ) (',' COMMENT )* '>'
+        UNIONTYPE '<' et_data_type ( ',' et_data_type ) '>' )
+    ;
+
+et_primitive_type
+    : TINYINT
+    | SMALLINT
+    | INT
+    | BIGINT
+    | BOOLEAN
+    | FLOAT
+    | DOUBLE
+    | STRING
+    | BINARY
+    | TIMESTAMP
+    | DECIMAL
+    ;
+
+et_location_specifier
+    : quoted_string
+    ;
+
+comments_oracle_loader
+    : RECORDS DELIMITED BY NEWLINE_
+    ;
+
+comments_oracle_datapump
+    : NOLOG
+    ;
+record_format_info
+    : ( RECORDS ( FIXED UNSIGNED_INTEGER
+                | VARIABLE UNSIGNED_INTEGER
+                | DELIMITED BY ( (DETECTED) NEWLINE_ | quoted_string )
+                | XMLTAG '('? quoted_string ( ',' quoted_string )* ')'? ))?
+      et_record_spec_options*?
```

Comment:
```suggestion
      et_record_spec_options*
```

Reply:
There may be other places where the rule is suffixed with `*?` and this is unnecessary since `*` implies 0 or more and therefore the subsequent `?` (implying optional) is unnecessary and redundant.